addons/account/views/account_journal_dashboard_view.xml

```diff
@@ -320,8 +320,8 @@
                             <div class="row" t-if="dashboard.number_draft">
                                 <div class="col overflow-hidden text-start">
                                     <a type="object" name="open_action" context="{'search_default_draft': '1'}">
-                                        <span t-if="journal_type == 'sale'" title="Invoices to Validate"><t t-out="dashboard.number_draft"/> Invoices to Validate</span>
-                                        <span t-if="journal_type == 'purchase'" title="Bills to Validate" id="account_dashboard_purchase_draft"><t t-out="dashboard.number_draft"/> Bills to Validate</span>
+                                        <span t-if="journal_type == 'sale'"><t t-out="dashboard.number_draft"/> Invoices to Validate</span>
+                                        <span t-if="journal_type == 'purchase'" id="account_dashboard_purchase_draft"><t t-out="dashboard.number_draft"/> Bills to Validate</span>
```

Comment:
Is there a reason to remove the `title`?
I think it can be useful on smaller screens, for instance:
![image](https://github.com/odoo/odoo/assets/44770049/d99fdc76-7444-4f1f-9796-1ab08aff3a6d)

Could be even worse with some languages

Reply:
I removed them after this this comment in your first review: https://github.com/odoo/odoo/pull/138188#discussion_r1360590483 . 

I'm putting them back waiting for your confirmation but I think it is indeed better to keep them.

Reply:
Yes, I probably got confused in the diff before, which was more complicated

Reply:
So you can put it back :)